doc/recipe/preprocessor.rst

```diff
@@ -180,12 +152,180 @@ steps:
 To get an overview on data fixes and how to implement new ones, please go to
 :ref:`fixing_data`.
 
-.. _Fx variables as cell measures or ancillary variables:
+.. _preprocessors_using_supplementary_variables:
+
+Ancillary variables and cell measures
```

Comment:
Should this say supplementary variables? In the documentations there is still a bit of back-and-forth between calling them `ancillary variables and cell measures` and `supplementary variables`. Maybe it would be easier to just refer to them as `supplementary` all the time?

Reply:
Good point. I changed it to more consistently use supplementary variables. I left some mentions of ancillary variables and cell measures because users may be more familiar with those.

Reply:
Thanks for clarifying!